src/gmail.js

```diff
@@ -394,8 +394,10 @@ class GmailAutoBccHandler {
 
         this.debug(this.formsEnabled);
         if (!this.formsEnabled[formElement.id]) {
-            this.debug("Form not found returning early");
-            return;
+            this.debug("Form not found. Trying again in 250ms");
+            setTimeout(() => {
+                this.updateCcAndBccRecipients(recipient, formElement);
+            }, 250);
```

Comment:
Lets return after the `setTimeout()` its non blocking and will execute the code below it which is dependent on the form. 

```js
  setTimeout(() => {
        this.updateCcAndBccRecipients(recipient, formElement);
     }, 250);
   return;
   ```

Reply:
Good call, sorry I missed that.
Done.